### DISCUSSION TITLE
Edit/Delete lists from the main tab only

### DISCUSSION THREAD

**Discussion by metastag** [community/community#8678] [Other Feature Feedback, Questions, & Ideas]
So currently we have to open a list first if we want to edit it's name/description, and i didn't see an option to delete it. Imo there should be a classic three-dots drop-down menu in the main lists tab, `(https://github.com/USSERNAME?tab=stars)`, where we have the option to edit/delete and perhaps any other options in the future.

And this goes without saying, but when deleting a list there must be a prompt for confirmation

**Comment by brianlovin** [comment]
Thanks for the note @MetaStag — I'll open an issue for us to think about how we can make the edit controls more clear. Right now you can delete a list by clicking the `Edit List` button in the header of the list view, It should appear here:

<img width="521" alt="Screen Shot 2021-12-10 at 16 13 37@2x" src="https://user-images.githubusercontent.com/1923260/145656133-3b77a55d-3f3b-469e-88eb-e7b25fb76d94.png">

And it will ask you to confirm:

<img width="496" alt="Screen Shot 2021-12-10 at 16 14 18@2x" src="https://user-images.githubusercontent.com/1923260/145656154-5b690b2b-ea79-403f-98e1-040cc17ace5e.png">

Thanks!

**Reply by praptosoehartono** [reply]
@Jokotingki01

**Comment by hr097** [comment]
is there any update guys on managing custom lists?
